test/CarouselSpec.js

```diff
@@ -238,4 +238,13 @@ describe('<Carousel>', () => {
     const wrapper = mount(<Carousel>{items}</Carousel>);
     wrapper.find('div').length.should.equal(4);
   });
+
+  it('Should move backwards if left control is clicked on the first item', () => {
+    const wrapper = mount(<Carousel defaultActiveIndex={0}>{items}</Carousel>);
+    wrapper.find('a.carousel-control-prev').simulate('click');
+
+    setTimeout(() => {
+      wrapper.assertSingle('.carousel-item-left');
```

Comment:
This assertion isn't actually going to fire, because the test case isn't async.

Perhaps make it async? Or else take a `done` callback in the fn.

Reply:
You could also use 
````
const clock = sinon.useFakeTimers({
      toFake: ['setTimeout'],
});
// some logic
clock.tick(100);
// check if X has happened
```

Reply:
Thanks! Turns out I planned the test case all wrong. I don't really see a good way to test this.